aspnet/mvc/views/layout.rst

```diff
@@ -0,0 +1,98 @@
+  :emphasize-lines: 42,66
+
+Specifying a Layout
+-------------------
+
+Razor views have a ``Layout`` property. Individual views specify a layout by defining this property:
+
+.. literalinclude:: ../../../common/samples/WebApplication1/src/WebApplication1/Views/_ViewStart.cshtml
+  :language: html
+  :emphasize-lines: 2
+
+The layout specified can use a full path (example: ``/Views/Shared/_Layout.cshtml``) or a partial name (example: ``_Layout``). When a partial name is provided, the :doc:`razor view engine <razor>` will search for the layout file using its standard discovery process. That is, the controller-associated folder is searched first, followed by the ``Shared`` folder. This discovery process is identical to the one used to discover :doc:`partial views <partial>`.
+
+Layouts can be nested; each layout can itself specify another layout. When rendering a view, :ref:`ViewStart <viewstart>` is run first, followed by the view itself, followed by its layout(s).
+
+Every layout must call ``RenderBody`` . Wherever the call to ``RenderBody`` is placed, the contents of the view will be rendered.
+
+Sections
+^^^^^^^^
+
+A layout can optionally reference one or more *sections*, by calling ``RenderSection``. Each call to ``RenderSection`` can specify whether that section is required or optional. Individual views specify the content to be rendered within a section using the ``@section`` razor syntax. If a view defines a section, it must be rendered (or an error will occur).
+
+An example ``@section`` definition in a view:
+
+.. literalinclude:: ../../../common/samples/WebApplication1/src/WebApplication1/Views/Manage/VerifyPhoneNumber.cshtml
+  :language: html
+  :lines: 28-30
+
+Sections only flow from views. They cannot be referenced from partials, view components, or other parts of the view system.
+
+.. _viewimports:
+
+Importing Shared Directives
+---------------------------
+
+Views can use razor directives to specify namespaces or perform :doc:`dependency injection <dependency-injection>`. Directives used by many views may be specified in a ``_ViewImports.cshtml`` file. The ``_ViewImports`` file supports the following directives:
+
+- addTagHelper
+- removeTagHelper
+- tagHelperPrefix
+- using
+- model
+- inherits
+- inject
+
+The file does not support other razor features, such as functions and section definitions.
+
+A sample ``_ViewImports`` file:
+
+.. literalinclude:: ../../../common/samples/WebApplication1/src/WebApplication1/Views/_ViewImports.cshtml
+  :language: html
+
+The ``_ViewImports`` file for an ASP.NET Core MVC app is typically placed in the ``Views`` folder root. A ``_ViewImports`` file can be placed within a controller-associated view folder, in which case it will only be applied to views within that folder. ``_ViewImports`` files are run first at the root level, and then for a ``_ViewImports`` file specified in the controller-associated folder, so settings specified at the root level may be overridden at the folder level.
```

Comment:
I almost feel like each of the 7 listed directives' merging/overriding behavior needs to be discussed individually:
* addTagHelper: All of them are added together; duplicates are ignored
* removeTagHelper: All of them run. Does order matter? What if I remove one and then later add it?
* tagHelperPrefix: closest to view wins?
* using: I assume these are all included, duplicates ignored.
* model: closest to view wins
* inherits: closest to view wins?
* inject: all are available?

Can you confirm? @NTaylorMullen